server.test.js

```diff
@@ -0,0 +1,7 @@
+//const server = require('./server');
```

Comment:
Will this be used later on?

Reply:
```suggestion
```

Reply:
Yes, opinions were divided on leaving it in or not because we'll need it soon.

Reply:
In that case, feel free to merge! 😊